### PR TITLE
Update from Weblate.

### PR DESCRIPTION
**Pull request by ubports-weblate**

### CODE REVIEW

File: po/de.po [hunk 1]
```diff
@@ -390,9 +390,10 @@ msgstr[0] "%1 Stunde"
 msgstr[1] "%1 Stunden"
 
 #: ../qml/RemindersModel.qml:74
+#, fuzzy
 msgid "%1 minute"
 msgid_plural "%1 minutes"
-msgstr[0] "%1 Minute"
+msgstr[0] "ist die Anzahl der Minuten"
```
Comment (Flohack74) [inline po/de.po]: I dont think this is correct.

Comment (NeoTheThird) [reply po/de.po]: Corrected it on translate.ubports.com.

Comment (NeoTheThird) [reply po/de.po]: You could either start the pr update manually or wait until Sunday for the new commit to show up.

Comment (Flohack74) [reply po/de.po]: Thx! Im afraid he translated 1 string and then told Weblate to substitute it globally, I found it in at least 3 other repos in the same way... meh

**Pull request merged**